guillotina/traversal.py

```diff
@@ -350,7 +354,7 @@ def set_root(self, root):
 
         await self.apply_authorization(request)
 
-        translator = queryMultiAdapter(
+        translator = query_multi_adapter(
```

Comment:
Do we have profiling information about the three multiadapters on the real_resolve ?

Reply:
Yes, we have some. I would be in favor of providing a different mechanism to hook into processing requests instead of doing translations on every request.

Reply:
I would be in favor of removing the translation framework since it's undocumented and unused.

We can do something in an addon.

Reply:
Its not only the translator aproach, also the view and renderer multiadapter. We should have an adapter so it can be implemented on addons without the need to modify real_traversal.